test/test_vmap.py

```diff
@@ -2106,16 +2106,41 @@ def test_slice(self):
              (torch.rand(3, 5, B0, B1, B2),), in_dims=2)
 
     def test_squeeze(self):
+        def verify_behavior(op, min_size=1):
```

Comment:
By size do you mean "number of dimensions"? If so we should name this `min_rank` or `min_ndim`.